Localization/en-US/home.po

```diff
@@ -0,0 +1,110 @@
+# Schema-Version: 1.0
+# URL: https://foldingathome.org/
+# Source-Language: en-US
+# Source-Version: 1.0
+# Source-Last-Modified: 20200705
+# Target-Language:
+# Target-Version:
+# Target-Last_Modified:
+
+msgid "I AM"
+msgstr ""
+
+msgid "One"
+msgstr ""
+
+msgid "IN A MILLION"
+msgstr ""
+
+msgid "Regardless if you are already folding or haven?t heard a word about it before, we need your help to reach our goal ? which is 1 million folders."
+msgstr ""
+
+msgid "Start folding now"
+msgstr ""
+
+msgid "[Hide this and watch the video.](#)"
+msgstr ""
+
+msgid "START FOLDING NOW"
+msgstr ""
+
+msgid "Find the version of the software you prefer and get started. Downloading Folding@home is completely free, easy to install and safe to use."
+msgstr ""
+
+msgid "Available for:"
+msgstr ""
+
+msgid "Linux"
+msgstr ""
+
+msgid "Windows"
+msgstr ""
+
+msgid "Mac"
+msgstr ""
+
+msgid "[Start folding](https://foldingathome.org/start-folding/)"
+msgstr ""
+
+msgid "Latest Posts"
+msgstr ""
+
+msgid "[The SARS-CoV-2 nucleocapsid protein is dynamic, disordered, and phase separates with RNA.](https://foldingathome.org/2020/06/26/the-sars-cov-2-nucleocapsid-protein-is-dynamic-disordered-and-phase-separates-with-rna/)"
```

Comment:
Should we really include these? They're probably auto-generated, I would guess.

Reply:
I guess we shouldn't.

Reply:
I removed it.